server/paymentProviders/stripe/webhook.ts

```diff
@@ -383,6 +437,28 @@ export const chargeDisputeClosed = async (event: Stripe.Event) => {
     return;
   }
 
+  if (chargeTransaction.Order) {
```

Comment:
Side effects should come after the actions, there are important checks and early-returns coming after this conditional.